h/services/jwt.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, jwt_signing_key):
 
     @classmethod
     def decode_oidc_idtoken(
-        cls, token: str, key_set_url: str, algorithms: list[str]
+        cls, token: str, keyset_url: str, algorithms: list[str]
```

Comment:
I couldn't resist renaming this for consistency with `keyset_url` and `KEYSET_URL` names used elsewhere. Also `key_set_url` just hurts my eyes, this would _absolutely_ be `keyset_url` in Pythonic naming style.

`keyset_url` makes it clear that it's the URL for the key set. Whereas `key_set_url` is more ambiguous. Does that mean `keyset_url` or `key_seturl` or indeed `key_set_url`? Is the `set` a verb or a noun? Etc.

Some programmers like to religiously separate all words with underscores in variable names but this is not good, readable or idiomatic naming style in Python.